index.html

```diff
@@ -80,17 +80,17 @@
 
           <div class="card row col-xs-5">
             <div class="info">
-              <h3>REVO</h3>
-              <p id="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+              <h4>REVO <small> Research of Electric Vehicles at Olin </small></h4>
+              <p id="description">A team that researches and builds electric vehicles at Olin. </p>
```

Comment:
nit: trailing space, but I'll approve this one and we can fix it later when we template.